src/github.com/getlantern/flashlight/feed/feed_test.go

```diff
@@ -45,7 +45,7 @@ func getFeed(t *testing.T, feedEndpoint string, locale string) {
 		"All feed items should be equal to total entries minus BuzzFeed entries")
 
 	for _, entry := range feed.Items["all"] {
-		assert.NotEmpty(t, entry.Description)
+		assert.NotEmpty(t, entry.Title)
```

Comment:
This test started failing for me today (there's occasionally articles without descriptions, depending on the source). A better test here is to assert an entry has a title.